application/controllers/AuthenticationController.php

```diff
@@ -64,6 +102,14 @@ public function loginAction()
     public function logoutAction()
     {
         $auth = $this->Auth();
+        if (isset($_COOKIE['remember-me'])) {
+            unset($_COOKIE['remember-me']);
+            $this->getResponse()->setCookie(
```

Comment:
I think it makes sense to introduce the `forgetMe()` function to the `Cookie` class. Upon calling `forgetMe()` the instance clones itself, sets the value to `null`, sets the expire time to `0` and returns the clone. Then you call it like this:

```php
$this->getResponse()->setCookie(
    (new RememberMeCookie())->forgetMe()
);
```